### ISSUE TITLE
Continuous Integration 

### ISSUE THREAD

**Issue by thom8047** [Eshop-project/CS-350#18]
Slight problem with the pylint. It seems it'd be more effective to have pylint used with everyone's Visual Studio Code via local repo. The GitHub CI, Python's pylint, is not effective in this case with many import errors. This led to some issues that would throw exceptions, which kept leading to Exit-1 returns and therefore we cannot be certain of what actually is causing the pylint error.

**Comment by ramosv** [comment]
As I was looking at the pylint and I've been getting some emails that tell me that pylint run filed. I don't know if anyone else in the team has been getting these same email. I will show you all on the next meeting

**Comment by thom8047** [comment]
Slight improve for the Continuous Integration. I updated some of the minor errors, and fixed the import error issues but now, when it's ran with the current format, pylint rated it 0.42/10.

**Comment by thom8047** [comment]
Every exception needs to be dealt with or disabled. There are probably about 20-30 different "grammatical" issues that the pylint caught and all of them were ridiculously tedious. Stupid errors like too many trailing white spaces, or comments in the form of milti-line strings: [""" *stuff* """]. I disabled most of the errors that I found to be useless or non-legitimate problems. This does not mean they should be over looked. Pylint follows PEP/8 documentation and therefore is what the best code should look like.
